src/components/base-address/BaseAddress.vue

```diff
@@ -169,8 +169,9 @@ import {
   useBaseValidations,
   spaceRules
 } from '@/components/base-address/factories'
-import { AddressIF, SchemaIF } from '@bcrs-shared-components/interfaces'
-import { AddressValidationRules } from '@bcrs-shared-components/enums'
+import { AddressIF } from '@bcrs-shared-components/interfaces'
+import { SchemaIF } from '@/interfaces'
```

Comment:
Can you import this from the shared folder?

In Storybook (ie, in this repo), this import will work because `@/interfaces` resolves to "./src/interfaces" (ie, local folder) [link to config](https://github.com/bcgov/bcrs-shared-components/blob/62a05e2e40024ec2cf6095285332f5811912071a/vite.config.ts#L19).

However, when this component is imported into another UI, `@/interfaces` will be that UI's local folder, which probably doesn't contain SchemaIF.

Reply:
I just changed that because it was throwing an error in hopes to solve the problem, I will change it back.

Reply:
Reverted. Thanks. Resolving.